e2e/scripts/smoke_test.sh

```diff
@@ -108,15 +132,15 @@ rook_infra::cleanup() {
 { #try
 
     rook_infra::create
-    sleep 5
+    sleep 10    #need to add code to retry docker info till success
```

Comment:
I didn't realize this check was here. I have removed it and we are making sure the container is in running state before we proceed.